### DISCUSSION TITLE
How do I remove these lines inbetween the colors and make them 'flat'

### DISCUSSION THREAD

**Discussion by bopchan** [fastfetch-cli/fastfetch#1094] [Q&A]
Sorry if this is a dumb question. 
![Screenshot 2024-07-14 072334](https://github.com/user-attachments/assets/d2757cf2-4f1f-4c4e-b59a-a6a014f7f1f7)

**Comment by CarterLi** [comment]
It depends on the terminal font you use. Some show the lines, some dont.

**Reply by bopchan** [reply]
Thank you
